hummingbot/data_feed/candles_feed/candles_base.py

```diff
@@ -264,16 +296,39 @@ def _get_rest_candles_headers(self):
         """
         pass
 
+    @property
+    def _is_last_candle_not_included_in_rest_request(self):
```

Comment:
I would suggest using `start`/`end` to match the current nomenclature and to use direct logic (remove `_not_`)

Reply:
yes it can be possible, sure, but I didn't get the start/end part

Reply:
Oh, just a cosmetic suggestion, the module uses `start_time` and `end_time`, end_candle and start_candle seem to match, but that's likely a matter of preference.

Reply:
suggestion accepted! i'll change it in the next 24hs

Reply:
Well, on the other hand the PR has been approved as-is, by both QA engineers. I am ok with simply making a note of it for the next update